Data/DbContext.cs

```diff
@@ -1,33 +1,40 @@
 using J3.Models;
-using J3.Models;
 using Microsoft.EntityFrameworkCore;
 
-namespace J3.Data
+namespace J3.Data;
+
+
+public class ColourContext : DbContext, IColourContext
 {
-    public class ColourContext : DbContext, IColourContext
+    public ColourContext(DbContextOptions<ColourContext> options)
+        : base(options) { }
+
+    public DbSet<Colour> Colours { get; set; }
+    public DbSet<User> Users { get; set; }
+    public DbSet<Collection> Collections { get; set; }
+    public DbSet<ColourCollection> ColourCollections { get; set; }
+    public DbSet<Comment> Comments { get; set; }       
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
-        public ColourContext(DbContextOptions<ColourContext> options)
-            : base(options) { }
-
-        public DbSet<Colour> Colours { get; set; }
-        public DbSet<User> Users { get; set; }
-        public DbSet<Collection> Collections { get; set; }
-        public DbSet<ColourCollection> ColourCollections { get; set; }
-        public DbSet<Comment> Comments { get; set; }
-
-        protected override void OnModelCreating(ModelBuilder modelBuilder)
-        {
-            base.OnModelCreating(modelBuilder);
-
-            modelBuilder.Entity<Collection>().ToTable("collection");
-            modelBuilder.Entity<ColourCollection>().ToTable("colour_collection");
-            modelBuilder.Entity<Colour>().ToTable("colour");
-            modelBuilder.Entity<Comment>().ToTable("comment");
-            modelBuilder.Entity<User>().ToTable("user");
-
-            modelBuilder
-                .Entity<ColourCollection>()
-                .HasKey(cc => new { cc.CollectionId, cc.ColourId });
-        }
+        base.OnModelCreating(modelBuilder);
+
+        modelBuilder.Entity<Colour>()
+            .ToTable("colour");
+        
+        modelBuilder.Entity<User>()
+            .ToTable("user");
+        
+        modelBuilder.Entity<Collection>()
+            .ToTable("collection");
+        
```

Comment:
I will look into how Max solved not having to do this OnModelCreating method and hopefully simplify this code a bit